jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/grid/AbstractTasksListGridPresenter.java

```diff
@@ -85,13 +86,13 @@
 
     }
 
-    private Constants constants = Constants.INSTANCE;
+    protected Constants constants = Constants.INSTANCE;
```

Comment:
Not the proper line to comment, but just to mention that this class would be named AbstractTaskListPresenter.